release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1050,6 +1050,11 @@ The Special Resource Operator (SRO) includes a consistent log output format with
 
 Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.
 
+[id="ocp-4-11-talm-upgrade-recovery"]
+====  Recovery of clusters after a failed upgrade
```

Comment:
Update is the preferred usage see  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update
```suggestion
====  Recovery of clusters after a failed update
```

Reply:
I would also recommend revising this subheading for present tense.